src/server/snap.rs

```diff
@@ -128,6 +129,7 @@ impl Stream for SnapChunk {
             n if n > SNAP_CHUNK_LEN => vec![0; SNAP_CHUNK_LEN],
             n => vec![0; n],
         };
+        let _with_io_type = WithIoType::new(self.io_type);
```

Comment:
```suggestion
        let _guard = WithIoType::new(self.io_type);
```
Better to use `_guard`.

Reply:
Other places use this naming, so let's keep it